src/transformers/generation/utils.py

```diff
@@ -4391,19 +4395,16 @@ def assisted_decoding(
                     # `new_token_len` can be 1 or 2 (next token in assistant + last token picked by the larger model)
                     new_token_len = candidate_input_ids.shape[1] - prev_seq_len
                     assist_inputs = candidate_input_ids[:, -new_token_len:]
-                    assist_attn = torch.ones_like(candidate_input_ids)
```

Comment:
Do we really need this @gante ? Allocating new memory here every time leads to some slow downs that are not insignificant for Distil Whisper

Reply:
Yes, this makes sense to remove, since it is the default attention mask! 👍